packages/rules-unit-testing/test/util.test.ts

```diff
@@ -165,7 +165,7 @@ describe('assertFails()', () => {
 
 describe('withFunctionTriggersDisabled()', () => {
   it('disabling function triggers does not throw, returns value', async function () {
-    const fetchSpy = sinon.spy(require('node-fetch'), 'default');
+    const fetchSpy = sinon.spy(globalThis, 'fetch');
```

Comment:
Does this work in Node 21.7.0+? https://github.com/firebase/firebase-js-sdk/pull/8492/files#diff-80382ef750022c8b0f79a66d4a5baf5fe0412100937528ef71e8938b62d312ffR22

Stub and mock won't, not 100% sure about spy but I think it also has to overwrite to spy. In any case, if there's a problem, I think what you gotta do is make an optional fetchImpl arg on withFunctionTriggersDisabled that defaults to `fetch` and then overwrite it here with a stub

Reply:
Looks like this is fine in Node now, resolving.